### PR TITLE
Check snoozed track for revoked proofs

### PR DESCRIPTION
**Pull request by patrickxb**
r? @maxtaco

### CODE REVIEW

File: go/libkb/rpc_exim.go [hunk 1]
```diff
@@ -109,7 +109,7 @@ func (ir IdentifyOutcome) ExportToUncheckedIdentity() *keybase1.Identity {
 	for j, d := range ir.Revoked {
 		// Should have all non-nil elements...
 		tmp.Revoked[j] = *ExportTrackDiff(d)
-		tmp.BreaksTracking = true
+		tmp.BreaksTracking = d.BreaksTracking()
```
Comment (maxtaco) [inline go/libkb/rpc_exim.go]: can't do this since it can set a `true` back to `false`

File: go/libkb/identify_state.go [hunk 2]
```diff
@@ -57,18 +57,35 @@ func (s *IdentifyState) computeRevokedProofs() {
...
+		s.res.RevokedDetails = append(s.res.RevokedDetails, ExportTrackIDComponentToRevokedProof(e))
+		if s.tmpTrack == nil {
+			s.res.Revoked = append(s.res.Revoked, TrackDiffRevoked{e})
+			continue
+		}
+
+		// There is a snoozed track in s.tmpTrack.
+		// The user could have snoozed the revoked proof already.
+		// Check s.tmpTrack to see if that is the case.
+		if s.tmpTrack.set.HasMember(e) {
```
Comment (maxtaco) [inline go/libkb/identify_state.go]: don't you want `!HasMember(e)` ??

File: go/libkb/identify_state.go [hunk 3]
```diff
@@ -57,18 +57,35 @@ func (s *IdentifyState) computeRevokedProofs() {
...
+			// proof was in snooze, too, so mark it as revoked.
```
Comment (maxtaco) [inline go/libkb/identify_state.go]: these comments seem backward but the code seems right

File: go/libkb/identify_state.go [hunk 4]
```diff
@@ -57,18 +57,35 @@ func (s *IdentifyState) computeRevokedProofs() {
...
 			s.res.Revoked = append(s.res.Revoked, TrackDiffRevoked{e})
-			s.res.RevokedDetails = append(s.res.RevokedDetails, ExportTrackIDComponentToRevokedProof(e))
+		} else {
+			// proof wasn't in snooze, so revoked proof already snoozed.
+			s.res.Revoked = append(s.res.Revoked, TrackDiffSnoozedRevoked{e})
```
Comment (maxtaco) [inline go/libkb/identify_state.go]: ok, i'm on board

**Comment by maxtaco** [comment]
Can you paste in a sample of what the U/I looks like from the CLI when this feature is put to use?

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 476bb2b4825a...654649b9f67a introduced some visual changes on win32:
 * Changed: **Profile-Unfollowed_Scrolled** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467919840861-476bb2b4825a-654649b9f67a-win32/Profile-Unfollowed_Scrolled-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467919840861-476bb2b4825a-654649b9f67a-win32/Profile-Unfollowed_Scrolled-654649b9f67a.png)  
   Profile-Unfollowed_Scrolled comparison
 * Changed: **Profile-Unfollowed_Profile_page** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467919840861-476bb2b4825a-654649b9f67a-win32/Profile-Unfollowed_Profile_page-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467919840861-476bb2b4825a-654649b9f67a-win32/Profile-Unfollowed_Profile_page-654649b9f67a.png)  
   Profile-Unfollowed_Profile_page comparison
 * Changed: **Profile-Unfollowed_Folders_Expanded** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467919840861-476bb2b4825a-654649b9f67a-win32/Profile-Unfollowed_Folders_Expanded-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467919840861-476bb2b4825a-654649b9f67a-win32/Profile-Unfollowed_Folders_Expanded-654649b9f67a.png)  
   Profile-Unfollowed_Folders_Expanded comparison
 * Changed: **Profile-Unfollowed_Few_Folders** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467919840861-476bb2b4825a-654649b9f67a-win32/Profile-Unfollowed_Few_Folders-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467919840861-476bb2b4825a-654649b9f67a-win32/Profile-Unfollowed_Few_Folders-654649b9f67a.png)  
   Profile-Unfollowed_Few_Folders comparison
 * Changed: **Profile-Unfollowed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467919840861-476bb2b4825a-654649b9f67a-win32/Profile-Unfollowed-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467919840861-476bb2b4825a-654649b9f67a-win32/Profile-Unfollowed-654649b9f67a.png)  
   Profile-Unfollowed comparison
 * Changed: **Profile-Followed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467919840861-476bb2b4825a-654649b9f67a-win32/Profile-Followed-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467919840861-476bb2b4825a-654649b9f67a-win32/Profile-Followed-654649b9f67a.png)  
   Profile-Followed comparison
 * Changed: **Profile-Changed_Scrolled** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467919840861-476bb2b4825a-654649b9f67a-win32/Profile-Changed_Scrolled-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467919840861-476bb2b4825a-654649b9f67a-win32/Profile-Changed_Scrolled-654649b9f67a.png)  
   Profile-Changed_Scrolled comparison
 * Changed: **Profile-Changed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467919840861-476bb2b4825a-654649b9f67a-win32/Profile-Changed-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467919840861-476bb2b4825a-654649b9f67a-win32/Profile-Changed-654649b9f67a.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1467919840861-476bb2b4825a-654649b9f67a-win32/Profile-Changed.png)

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 476bb2b4825a...1982fd58879c introduced some visual changes on linux:
 * Changed: **Profile-Unfollowed_Scrolled** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467919884533-476bb2b4825a-1982fd58879c-linux/Profile-Unfollowed_Scrolled-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467919884533-476bb2b4825a-1982fd58879c-linux/Profile-Unfollowed_Scrolled-1982fd58879c.png)  
   Profile-Unfollowed_Scrolled comparison
 * Changed: **Profile-Unfollowed_Profile_page** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467919884533-476bb2b4825a-1982fd58879c-linux/Profile-Unfollowed_Profile_page-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467919884533-476bb2b4825a-1982fd58879c-linux/Profile-Unfollowed_Profile_page-1982fd58879c.png)  
   Profile-Unfollowed_Profile_page comparison
 * Changed: **Profile-Unfollowed_Folders_Expanded** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467919884533-476bb2b4825a-1982fd58879c-linux/Profile-Unfollowed_Folders_Expanded-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467919884533-476bb2b4825a-1982fd58879c-linux/Profile-Unfollowed_Folders_Expanded-1982fd58879c.png)  
   Profile-Unfollowed_Folders_Expanded comparison
 * Changed: **Profile-Unfollowed_Few_Folders** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467919884533-476bb2b4825a-1982fd58879c-linux/Profile-Unfollowed_Few_Folders-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467919884533-476bb2b4825a-1982fd58879c-linux/Profile-Unfollowed_Few_Folders-1982fd58879c.png)  
   Profile-Unfollowed_Few_Folders comparison
 * Changed: **Profile-Unfollowed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467919884533-476bb2b4825a-1982fd58879c-linux/Profile-Unfollowed-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467919884533-476bb2b4825a-1982fd58879c-linux/Profile-Unfollowed-1982fd58879c.png)  
   Profile-Unfollowed comparison
 * Changed: **Profile-Followed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467919884533-476bb2b4825a-1982fd58879c-linux/Profile-Followed-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467919884533-476bb2b4825a-1982fd58879c-linux/Profile-Followed-1982fd58879c.png)  
   Profile-Followed comparison
 * Changed: **Profile-Changed_Scrolled** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467919884533-476bb2b4825a-1982fd58879c-linux/Profile-Changed_Scrolled-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467919884533-476bb2b4825a-1982fd58879c-linux/Profile-Changed_Scrolled-1982fd58879c.png)  
   Profile-Changed_Scrolled comparison
 * Changed: **Profile-Changed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467919884533-476bb2b4825a-1982fd58879c-linux/Profile-Changed-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467919884533-476bb2b4825a-1982fd58879c-linux/Profile-Changed-1982fd58879c.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1467919884533-476bb2b4825a-1982fd58879c-linux/Profile-Changed.png)

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 476bb2b4825a...042439664b6d introduced some visual changes on win32:
 * Changed: **Profile-Unfollowed_Scrolled** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467922179046-476bb2b4825a-042439664b6d-win32/Profile-Unfollowed_Scrolled-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467922179046-476bb2b4825a-042439664b6d-win32/Profile-Unfollowed_Scrolled-042439664b6d.png)  
   Profile-Unfollowed_Scrolled comparison
 * Changed: **Profile-Unfollowed_Profile_page** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467922179046-476bb2b4825a-042439664b6d-win32/Profile-Unfollowed_Profile_page-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467922179046-476bb2b4825a-042439664b6d-win32/Profile-Unfollowed_Profile_page-042439664b6d.png)  
   Profile-Unfollowed_Profile_page comparison
 * Changed: **Profile-Unfollowed_Folders_Expanded** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467922179046-476bb2b4825a-042439664b6d-win32/Profile-Unfollowed_Folders_Expanded-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467922179046-476bb2b4825a-042439664b6d-win32/Profile-Unfollowed_Folders_Expanded-042439664b6d.png)  
   Profile-Unfollowed_Folders_Expanded comparison
 * Changed: **Profile-Unfollowed_Few_Folders** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467922179046-476bb2b4825a-042439664b6d-win32/Profile-Unfollowed_Few_Folders-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467922179046-476bb2b4825a-042439664b6d-win32/Profile-Unfollowed_Few_Folders-042439664b6d.png)  
   Profile-Unfollowed_Few_Folders comparison
 * Changed: **Profile-Unfollowed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467922179046-476bb2b4825a-042439664b6d-win32/Profile-Unfollowed-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467922179046-476bb2b4825a-042439664b6d-win32/Profile-Unfollowed-042439664b6d.png)  
   Profile-Unfollowed comparison
 * Changed: **Profile-Followed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467922179046-476bb2b4825a-042439664b6d-win32/Profile-Followed-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467922179046-476bb2b4825a-042439664b6d-win32/Profile-Followed-042439664b6d.png)  
   Profile-Followed comparison
 * Changed: **Profile-Changed_Scrolled** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467922179046-476bb2b4825a-042439664b6d-win32/Profile-Changed_Scrolled-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467922179046-476bb2b4825a-042439664b6d-win32/Profile-Changed_Scrolled-042439664b6d.png)  
   Profile-Changed_Scrolled comparison
 * Changed: **Profile-Changed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467922179046-476bb2b4825a-042439664b6d-win32/Profile-Changed-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467922179046-476bb2b4825a-042439664b6d-win32/Profile-Changed-042439664b6d.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1467922179046-476bb2b4825a-042439664b6d-win32/Profile-Changed.png)

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 476bb2b4825a...cf5632a53921 introduced some visual changes on linux:
 * Changed: **Profile-Unfollowed_Scrolled** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467922272397-476bb2b4825a-cf5632a53921-linux/Profile-Unfollowed_Scrolled-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467922272397-476bb2b4825a-cf5632a53921-linux/Profile-Unfollowed_Scrolled-cf5632a53921.png)  
   Profile-Unfollowed_Scrolled comparison
 * Changed: **Profile-Unfollowed_Profile_page** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467922272397-476bb2b4825a-cf5632a53921-linux/Profile-Unfollowed_Profile_page-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467922272397-476bb2b4825a-cf5632a53921-linux/Profile-Unfollowed_Profile_page-cf5632a53921.png)  
   Profile-Unfollowed_Profile_page comparison
 * Changed: **Profile-Unfollowed_Folders_Expanded** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467922272397-476bb2b4825a-cf5632a53921-linux/Profile-Unfollowed_Folders_Expanded-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467922272397-476bb2b4825a-cf5632a53921-linux/Profile-Unfollowed_Folders_Expanded-cf5632a53921.png)  
   Profile-Unfollowed_Folders_Expanded comparison
 * Changed: **Profile-Unfollowed_Few_Folders** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467922272397-476bb2b4825a-cf5632a53921-linux/Profile-Unfollowed_Few_Folders-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467922272397-476bb2b4825a-cf5632a53921-linux/Profile-Unfollowed_Few_Folders-cf5632a53921.png)  
   Profile-Unfollowed_Few_Folders comparison
 * Changed: **Profile-Unfollowed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467922272397-476bb2b4825a-cf5632a53921-linux/Profile-Unfollowed-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467922272397-476bb2b4825a-cf5632a53921-linux/Profile-Unfollowed-cf5632a53921.png)  
   Profile-Unfollowed comparison
 * Changed: **Profile-Followed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467922272397-476bb2b4825a-cf5632a53921-linux/Profile-Followed-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467922272397-476bb2b4825a-cf5632a53921-linux/Profile-Followed-cf5632a53921.png)  
   Profile-Followed comparison
 * Changed: **Profile-Changed_Scrolled** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467922272397-476bb2b4825a-cf5632a53921-linux/Profile-Changed_Scrolled-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467922272397-476bb2b4825a-cf5632a53921-linux/Profile-Changed_Scrolled-cf5632a53921.png)  
   Profile-Changed_Scrolled comparison
 * Changed: **Profile-Changed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467922272397-476bb2b4825a-cf5632a53921-linux/Profile-Changed-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467922272397-476bb2b4825a-cf5632a53921-linux/Profile-Changed-cf5632a53921.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1467922272397-476bb2b4825a-cf5632a53921-linux/Profile-Changed.png)

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 476bb2b4825a...bb17a0ceea6d introduced some visual changes on win32:
 * Changed: **Profile-Unfollowed_Scrolled** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467923276100-476bb2b4825a-bb17a0ceea6d-win32/Profile-Unfollowed_Scrolled-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467923276100-476bb2b4825a-bb17a0ceea6d-win32/Profile-Unfollowed_Scrolled-bb17a0ceea6d.png)  
   Profile-Unfollowed_Scrolled comparison
 * Changed: **Profile-Unfollowed_Profile_page** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467923276100-476bb2b4825a-bb17a0ceea6d-win32/Profile-Unfollowed_Profile_page-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467923276100-476bb2b4825a-bb17a0ceea6d-win32/Profile-Unfollowed_Profile_page-bb17a0ceea6d.png)  
   Profile-Unfollowed_Profile_page comparison
 * Changed: **Profile-Unfollowed_Folders_Expanded** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467923276100-476bb2b4825a-bb17a0ceea6d-win32/Profile-Unfollowed_Folders_Expanded-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467923276100-476bb2b4825a-bb17a0ceea6d-win32/Profile-Unfollowed_Folders_Expanded-bb17a0ceea6d.png)  
   Profile-Unfollowed_Folders_Expanded comparison
 * Changed: **Profile-Unfollowed_Few_Folders** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467923276100-476bb2b4825a-bb17a0ceea6d-win32/Profile-Unfollowed_Few_Folders-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467923276100-476bb2b4825a-bb17a0ceea6d-win32/Profile-Unfollowed_Few_Folders-bb17a0ceea6d.png)  
   Profile-Unfollowed_Few_Folders comparison
 * Changed: **Profile-Unfollowed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467923276100-476bb2b4825a-bb17a0ceea6d-win32/Profile-Unfollowed-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467923276100-476bb2b4825a-bb17a0ceea6d-win32/Profile-Unfollowed-bb17a0ceea6d.png)  
   Profile-Unfollowed comparison
 * Changed: **Profile-Followed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467923276100-476bb2b4825a-bb17a0ceea6d-win32/Profile-Followed-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467923276100-476bb2b4825a-bb17a0ceea6d-win32/Profile-Followed-bb17a0ceea6d.png)  
   Profile-Followed comparison
 * Changed: **Profile-Changed_Scrolled** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467923276100-476bb2b4825a-bb17a0ceea6d-win32/Profile-Changed_Scrolled-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467923276100-476bb2b4825a-bb17a0ceea6d-win32/Profile-Changed_Scrolled-bb17a0ceea6d.png)  
   Profile-Changed_Scrolled comparison
 * Changed: **Profile-Changed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467923276100-476bb2b4825a-bb17a0ceea6d-win32/Profile-Changed-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467923276100-476bb2b4825a-bb17a0ceea6d-win32/Profile-Changed-bb17a0ceea6d.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1467923276100-476bb2b4825a-bb17a0ceea6d-win32/Profile-Changed.png)

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 476bb2b4825a...bf2b545af3c6 introduced some visual changes on linux:
 * Changed: **Profile-Unfollowed_Scrolled** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467923418153-476bb2b4825a-bf2b545af3c6-linux/Profile-Unfollowed_Scrolled-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467923418153-476bb2b4825a-bf2b545af3c6-linux/Profile-Unfollowed_Scrolled-bf2b545af3c6.png)  
   Profile-Unfollowed_Scrolled comparison
 * Changed: **Profile-Unfollowed_Profile_page** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467923418153-476bb2b4825a-bf2b545af3c6-linux/Profile-Unfollowed_Profile_page-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467923418153-476bb2b4825a-bf2b545af3c6-linux/Profile-Unfollowed_Profile_page-bf2b545af3c6.png)  
   Profile-Unfollowed_Profile_page comparison
 * Changed: **Profile-Unfollowed_Folders_Expanded** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467923418153-476bb2b4825a-bf2b545af3c6-linux/Profile-Unfollowed_Folders_Expanded-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467923418153-476bb2b4825a-bf2b545af3c6-linux/Profile-Unfollowed_Folders_Expanded-bf2b545af3c6.png)  
   Profile-Unfollowed_Folders_Expanded comparison
 * Changed: **Profile-Unfollowed_Few_Folders** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467923418153-476bb2b4825a-bf2b545af3c6-linux/Profile-Unfollowed_Few_Folders-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467923418153-476bb2b4825a-bf2b545af3c6-linux/Profile-Unfollowed_Few_Folders-bf2b545af3c6.png)  
   Profile-Unfollowed_Few_Folders comparison
 * Changed: **Profile-Unfollowed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467923418153-476bb2b4825a-bf2b545af3c6-linux/Profile-Unfollowed-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467923418153-476bb2b4825a-bf2b545af3c6-linux/Profile-Unfollowed-bf2b545af3c6.png)  
   Profile-Unfollowed comparison
 * Changed: **Profile-Followed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467923418153-476bb2b4825a-bf2b545af3c6-linux/Profile-Followed-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467923418153-476bb2b4825a-bf2b545af3c6-linux/Profile-Followed-bf2b545af3c6.png)  
   Profile-Followed comparison
 * Changed: **Profile-Changed_Scrolled** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467923418153-476bb2b4825a-bf2b545af3c6-linux/Profile-Changed_Scrolled-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467923418153-476bb2b4825a-bf2b545af3c6-linux/Profile-Changed_Scrolled-bf2b545af3c6.png)  
   Profile-Changed_Scrolled comparison
 * Changed: **Profile-Changed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467923418153-476bb2b4825a-bf2b545af3c6-linux/Profile-Changed-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467923418153-476bb2b4825a-bf2b545af3c6-linux/Profile-Changed-bf2b545af3c6.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1467923418153-476bb2b4825a-bf2b545af3c6-linux/Profile-Changed.png)

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 476bb2b4825a...bf2b545af3c6 introduced some visual changes on linux:
 * Changed: **Profile-Unfollowed_Scrolled** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467926672553-476bb2b4825a-bf2b545af3c6-linux/Profile-Unfollowed_Scrolled-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467926672553-476bb2b4825a-bf2b545af3c6-linux/Profile-Unfollowed_Scrolled-bf2b545af3c6.png)  
   Profile-Unfollowed_Scrolled comparison
 * Changed: **Profile-Unfollowed_Profile_page** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467926672553-476bb2b4825a-bf2b545af3c6-linux/Profile-Unfollowed_Profile_page-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467926672553-476bb2b4825a-bf2b545af3c6-linux/Profile-Unfollowed_Profile_page-bf2b545af3c6.png)  
   Profile-Unfollowed_Profile_page comparison
 * Changed: **Profile-Unfollowed_Folders_Expanded** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467926672553-476bb2b4825a-bf2b545af3c6-linux/Profile-Unfollowed_Folders_Expanded-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467926672553-476bb2b4825a-bf2b545af3c6-linux/Profile-Unfollowed_Folders_Expanded-bf2b545af3c6.png)  
   Profile-Unfollowed_Folders_Expanded comparison
 * Changed: **Profile-Unfollowed_Few_Folders** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467926672553-476bb2b4825a-bf2b545af3c6-linux/Profile-Unfollowed_Few_Folders-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467926672553-476bb2b4825a-bf2b545af3c6-linux/Profile-Unfollowed_Few_Folders-bf2b545af3c6.png)  
   Profile-Unfollowed_Few_Folders comparison
 * Changed: **Profile-Unfollowed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467926672553-476bb2b4825a-bf2b545af3c6-linux/Profile-Unfollowed-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467926672553-476bb2b4825a-bf2b545af3c6-linux/Profile-Unfollowed-bf2b545af3c6.png)  
   Profile-Unfollowed comparison
 * Changed: **Profile-Followed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467926672553-476bb2b4825a-bf2b545af3c6-linux/Profile-Followed-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467926672553-476bb2b4825a-bf2b545af3c6-linux/Profile-Followed-bf2b545af3c6.png)  
   Profile-Followed comparison
 * Changed: **Profile-Changed_Scrolled** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467926672553-476bb2b4825a-bf2b545af3c6-linux/Profile-Changed_Scrolled-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467926672553-476bb2b4825a-bf2b545af3c6-linux/Profile-Changed_Scrolled-bf2b545af3c6.png)  
   Profile-Changed_Scrolled comparison
 * Changed: **Profile-Changed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1467926672553-476bb2b4825a-bf2b545af3c6-linux/Profile-Changed-476bb2b4825a.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1467926672553-476bb2b4825a-bf2b545af3c6-linux/Profile-Changed-bf2b545af3c6.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1467926672553-476bb2b4825a-bf2b545af3c6-linux/Profile-Changed.png)

**Comment by patrickxb** [comment]
Here's what it looks like to id a user with a snoozed revoked proof:

~/src/github.com/keybase/client/go> altkbcl -d=0 id a123                                                                                                        
▶ WARNING Running in devel mode
▶ INFO Identifying a123
You last tracked a123 on 2016-07-06 12:16:22 CDT
▶ WARNING Some proofs you previously tracked were revoked:
✖ <Deleted proof: a123@rooter (snoozed)>

**Comment by maxtaco** [comment]
is that a bad copy-paste, or is the output bad?

**Comment by patrickxb** [comment]
Let me try again:

    ~/src/github.com/keybase/client/go> altkbcl -d=0 id a123         
    ▶ WARNING Running in devel mode
    ▶ INFO Identifying a123
    You last tracked a123 on 2016-07-06 12:16:22 CDT
    ▶ WARNING Some proofs you previously tracked were revoked:
    ✖ <Deleted proof: a123@rooter (snoozed)>

**Comment by maxtaco** [comment]
👍
